iceoryx_utils/platform/win/include/iceoryx_utils/platform/acl.hpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
@elfenpiff  will we use this at all on Windows? Or will the acl functionality be skipped on a higher level?

Reply:
I wanted to keep the ability to use this in Windows. In Windows we have something called DACLs (https://docs.microsoft.com/en-us/windows/win32/secauthz/dacls-and-aces) and I think it is possible to map these ACL calls to Windows.

If in the future the need arises to have some more security in Windows we still have the possibility to go that road without changing our actual code base.